src/mutation.ts

```diff
@@ -154,8 +196,9 @@ export class Mutation {
    * // ]
    * @private
    */
-  static encodeSetCell(data) {
-    const mutations: any[] = [];
+  static encodeSetCell(data: Metadata): EncodeSetCellResponse[] {
```

Comment:
Can we use [`ISetCell`](https://github.com/googleapis/nodejs-bigtable/blob/master/proto/bigtable.d.ts#L3228) here?